manifests/e/e2eSoft/e2eSoftiVCam/6.2.2.0/e2eSoft.e2eSoftiVCam.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: e2eSoft.e2eSoftiVCam

```

Let's remove the publisher from here.

```suggestion
PackageIdentifier: e2eSoft.iVCam
```